### DISCUSSION TITLE
Idproxy login

### DISCUSSION THREAD

**Discussion by ahmadharis14** [ldproxy/ldproxy#673] [Setup] [answered]
While deploying the Idproxy, the default webpage of "http://localhost:7080/manager/" to first default log in with userid: admin and password: admin, when I change the password - it doesn't accept it. Only one time after hours of struggle - a new interface of log in came but it would not accept the log in. I am stuck here for days, any help would be very appreciated.

**Comment by azahnen** [comment, marked as answer]
I investigated your issue and found out it was a bug in ldproxy v3.2.x, see #687. The issue is fixed in `iide/ldproxy:next`. To make sure that it works for you, you should delete all cookies for `localhost:7080`.

**Reply by ahmadharis14** [reply]
Thanks for the feedback :)
